sky/data/storage.py

```diff
@@ -2457,6 +2616,10 @@ def delete(self) -> None:
         logger.info(f'{colorama.Fore.GREEN}{msg_str}'
                     f'{colorama.Style.RESET_ALL}')
 
+    def remove_objects_from_sub_path(self) -> None:
+        assert self._bucket_sub_path is not None, 'bucket_sub_path is not set'
+        raise NotImplementedError('Not implemented')
```

Comment:
AWS and GCS use command-line interfaces, and the official documentation is quite clear and easy to follow.

AZ doesn't use the CLI client. Our codebase currently uses this SDK, but I didn't find a directory delete API in the [doc](https://learn.microsoft.com/en-us/python/api/azure-mgmt-storage/azure.mgmt.storage.storagemanagementclient?view=azure-python), could u help @romilbhardwaj @Michaelvll

Reply:
Maybe I can use the [CLI](https://learn.microsoft.com/en-us/cli/azure/storage/blob?view=azure-cli-latest#az-storage-blob-delete-batch), why does the code base use SDK?